lib/xcode/install/simulators.rb

```diff
@@ -10,7 +10,8 @@ def self.options
         [['--install=name', 'Install simulator beginning with name, e.g. \'iOS 8.4\', \'tvOS 9.0\'.'],
          ['--force', 'Install even if the same version is already installed.'],
          ['--no-install', 'Only download DMG, but do not install it.'],
-         ['--no-progress', 'Don’t show download progress.']].concat(super)
+         ['--no-progress', 'Don’t show download progress.'],
+         ['--retry-download-count', 'Count of retrying download when curl is failed.']].concat(super)
```

Comment:
```suggestion
         ['--retry-download-count', 'Count of retrying download when the curl operation fails.']].concat(super)
```

Reply:
It is my English error from https://github.com/xcpretty/xcode-install/pull/400/files#diff-aa566aea6065490b7209721b920e3410R21, could you help to fix it? sorry

Reply:
What about changing the variable and the explanation to this:
```bash
'--number-of-try', 'How many times try to download DMG file if downloading fails. Default is 3.'
```